### PR TITLE
(SIMP-2354) Update simplib refs

### PR DESCRIPTION
**Pull request by op-ct**
This patch cleans up after some major releases in its dependencies:

All dependencies needed to build and test this version of the module
have been tagged in `.fixtures.yml` or recorded in `Gemfile.lock`

SIMP-2355 #close #comment Removed simplib refs
SIMP-2348 #close #comment removed PAM refs
SIMP-2413 #close #comment locked `.fixtures` and `Gemfile.lock`

### CODE REVIEW

**Comment by jeefberkey** [comment]
How do I use fixtures.tracking?

**Comment by op-ct** [comment]
@jeefberkey `.fixtures.tracking` maps what branches in each fixture repository track progress for each tagged repo in `.fixtures.yml`.  You can use this information to check that branch for more recent tags.  

Ideally, an automated task should use this file for that purpose in the near future.

**Comment by op-ct** [comment]
@jeefberkey On second thought: it's a little premature to anticipate the needs of a hypothetical tool, so I've removed `.fixtures.tracking`.  It seems a lot more flexible (and less of a hassle) to just look up the tagged fixtures' branches in the `Puppetfile.tracking` from a target ref of **simp-core**.

File: .gitignore [hunk 1]
```diff
@@ -9,7 +9,6 @@ dist
 /.rspec_system
 /.vagrant
 /.bundle
-/Gemfile.lock
```
Comment (nick-markowski) [inline .gitignore]: Why so?

**Pull request merged**